apps/stix-shifter/README.md

```diff
@@ -40,6 +40,19 @@ make dev-mode
 stix-shifter-threatbus -c config.yaml
 ```
 
+### Docker
+
+You can also run this app via Docker.
+
+- Build it:
+  ```
+  docker build . -t tenzir/stix-shifter-threatbus:latest
+  ```
+- Run it (and mount a custom config file into the container):
+  ```
+  docker run --net=host -v /path/to/your/conf.yaml:/opt/tenzir/threatbus/stix-shifter-threatbus/config.yaml
```

Comment:
```suggestion
  docker run --net=host -v /path/to/your/conf.yaml:/opt/tenzir/threatbus/stix-shifter-threatbus/ tenzir/stix-shifter-threatbus
```

Otherwise this doesn't work for me: You need to specify a directory to copy the file into and you must specify a container image to run.

Reply:
As discussed via call and Slack, this is not necessary.

Reply:
This is not resolved, you did not add the container name. As-is, this is not running at all.